contracts/Liquidation.sol

```diff
@@ -196,6 +198,30 @@ contract Liquidation is ILiquidation, Ownable {
         return amountToEscrow;
     }
 
+    /**
+     * @return true if the margin is greater than 10x liquidation gas cost (in quote tokens)
+     * @param liquidatedBaseChange How much base token to be liquidated
+     * @param liquidatedQuoteChange How much quote token to be liquidated
+     * @param balanceToBeLiquidated The balance of account to be liquidated
+     */
+    function checkPartialLiquidation(
```

Comment:
Should accept a `Balances.Position`.

Reply:
Yep good catches, I've changed it to `Balances.Position`. I could have chucked the updated position in the `Balances.Account` param but decided not to because that'll get confusing pretty quickly since that uses the `lastUpdatedGasPrice` of the *old* account balance so don't want to overlap the two